build.gradle

```diff
@@ -103,6 +103,8 @@ subprojects {
         testImplementation 'org.junit.vintage:junit-vintage-engine'
         testImplementation "org.mockito:mockito-core:${versionMap.mockito}"
         testImplementation "org.mockito:mockito-junit-jupiter:${versionMap.mockito}"
+        testImplementation 'org.hamcrest:hamcrest:2.2'
+        testImplementation 'org.awaitility:awaitility:4.2.0'
```

Comment:
This isn't used by every sub project. What's the threshold for moving dependencies to the root of the project?

Reply:
I take it you are referring to Awaitility, and that is correct. I think this is OK in this scenario since this is a test dependency. It is only needed when running Gradle tests, and won't be part of the final runtime. The reason I included it is because it is the primary tool Data Prepper is using for asynchronous testing. So if a project ends up need it, it is already there. I can revert the Awaitility if you prefer.

Regarding Hamcrest, I'd like to see more tests using it. :)

Reply:
Also, part of my motivation here is to reduce the number of Dependabot PRs this project gets.

Reply:
Ahh, sounds good. I am all for it now.